config/jobs/kubernetes/sig-security/cvelist-public.yaml

```diff
@@ -0,0 +1,26 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: validate-cve-files
+    decorate: true
+    always_run: false
+    optional: true
+    run_if_changed: 'CVE.*json$'
+    spec:
+      containers:
+      - image: golang
+        command:
+        - /bin/bash
+        args:
+        - -c
+        - |
+          set -euo; \
+          files=( $(git diff --name-only --diff-filter=d $PULL_BASE_SHA...$PULL_PULL_SHA | grep "\/CVE.*json$") ); \
+          for f in ${files[@]}:; \
```

Comment:
If you want to just pass in all the files, I'll update the validate script to take multiple files.

Reply:
@tallclair let’s leave it as is now, (one file per invocation of the shell script). once we have the skeleton working, we can tweak

Reply:
up to you. I already merged the updated script. The advantage being that it will check all files before failing.

Reply:
fixed now.